rtsf-at-checkout-device-scale/Dockerfile

```diff
@@ -12,21 +12,22 @@ LABEL license='SPDX-License-Identifier: Apache-2.0' \
 RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
 
 # add git for go modules
-RUN apk update && apk add make git
+RUN apk update && apk add --no-cache make git gcc libc-dev zeromq-dev linux-headers
```

Comment:
We can avoid `gcc libc-dev zeromq-dev linux-headers`  if we add the `no_zmq` tag to the build.